tools/versioner/cmd/unstage.go

```diff
@@ -0,0 +1,355 @@
+package cmd
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"github.com/Azure/azure-sdk-for-go/tools/apidiff/repo"
+	"github.com/Azure/azure-sdk-for-go/tools/internal/modinfo"
+	"github.com/Masterminds/semver"
+	"github.com/spf13/cobra"
+	"io"
+	"os"
+	"path/filepath"
+	"regexp"
+	"sort"
+	"strings"
+)
+
+var unstageCmd = &cobra.Command{
```

Comment:
While I'm not opposed to moving this content to `unstage.go` the problem is any code changes made here don't show up in the diff.  If you really want to move this content can you please do it in a separate PR that introduces zero code changes?

Reply:
How about I put this command in the same file with rootCmd, therefore the diff will be calculate normally. And after reviewed, they can go in separate files

Reply:
I move these changes to [another pr](https://github.com/Azure/azure-sdk-for-go/pull/5981)